examples/gps_simpletest.py

```diff
@@ -17,7 +17,7 @@
 # For other boards set RX = GPS module TX, and TX = GPS module RX pins.
 rx = board.RX  # Change to board.GP4 for Raspberry Pi Pico boards
```

Comment:
Looking at the Pico pinouts -- these comments should be corrected
GP4 is TX1 and GP5 is RX1 so if the change is made to correct have the calling arguments as (tx,rx...) then then for the PICO, it should  tx = board.GP4 and rx = board.GP5

Reply:
Thanks for catching this. I fixed the pin suggestions and clarified the TX-to-RX and RX-to-TX comment.